build/embed.bzl

```diff
@@ -143,19 +158,20 @@ def _cc_embed(ctx):
       subs = {
         "start": "_binary_{}_start".format(clean),
         "end": "_binary_{}_end".format(clean),
-        "var": _clean_string(i.path.split("/")[-1]),
+        "var": _clean_string(_strip_extensions(i.path.split("/")[-1], ctx.attr.strip)),
       }
 
       symbols.append("extern const char {start};".format(**subs))
       symbols.append("extern const char {end};".format(**subs))
       accessors.append("inline const std::string_view {var}(&{start}, &{end} - &{start});".format(**subs))
 
-    ifguard = _clean_path(hfile.path).upper()
+    ifguard = _clean_path(hfile.path).upper() + "_"
```

Comment:
add a comment here on what ifguard means - i am assuming this is a "header define guard"
?

Reply:
done